### PR TITLE
[#19] Add base comments

### PR DESCRIPTION
**Pull request by Voovaaa**

### CODE REVIEW

File: first_django_project/discography/models.py [hunk 1]
```diff
@@ -11,10 +10,20 @@ def __str__(self):
...
     date_published = models.DateField()
     artist = models.ForeignKey(Artist, on_delete=models.CASCADE, related_name="tracks")
 
-
     def __str__(self):
         return f"{self.name}, artist - {self.artist}"
+
+
+class Comment(models.Model):
+    comment_text = models.CharField(max_length=1000,
```
Comment (Safintim) [inline first_django_project/discography/models.py]: simple text. redundant comment. comment.comment_text?

File: first_django_project/discography/models.py [hunk 2]
```diff
@@ -11,10 +10,20 @@ def __str__(self):
...
+                                    validators=[MinLengthValidator(3, message="Comment's len must be more than 2")])
+    date_published = models.DateTimeField(auto_now=True)
```
Comment (Safintim) [inline first_django_project/discography/models.py]: Use auto_now_add=True. auto_now for modified_at

File: first_django_project/discography/views.py [hunk 3]
```diff
@@ -10,9 +10,11 @@ class IndexView(generic.ListView):
     model = Artist
 
 
-class ArtistDetailView(generic.DetailView):
+class ArtistDetailView(generic.CreateView, generic.DetailView):
     template_name = 'discography/artist_detail.html'
     model = Artist
+    form_class = CommentForm
+    success_url = "/"
```
Comment (Safintim) [inline first_django_project/discography/views.py]: reverse to /1 (artist_detail)

File: first_django_project/discography/migrations/0003_alter_track_name.py [hunk 4]
```diff
@@ -1,19 +0,0 @@
-# Generated by Django 4.0.4 on 2022-06-01 12:16
```
Comment (Safintim) [inline first_django_project/discography/migrations/0003_alter_track_name.py:1]: Delete this migration

File: first_django_project/discography/migrations/0002_alter_track_artist.py [hunk 5]
```diff
@@ -1,19 +0,0 @@
-# Generated by Django 4.0.4 on 2022-05-28 14:03
```
Comment (Safintim) [inline first_django_project/discography/migrations/0002_alter_track_artist.py:1]: Delete this migration

File: first_django_project/discography/models.py (hunk 1 above)
Comment (Voovaaa) [reply first_django_project/discography/models.py]: like in official tutorial 
image

File: first_django_project/discography/views.py (hunk 3 above)
Comment (Voovaaa) [reply first_django_project/discography/views.py]: ? how

File: first_django_project/discography/migrations/0003_alter_track_name.py (hunk 4 above)
Comment (Voovaaa) [reply first_django_project/discography/migrations/0003_alter_track_name.py:1]: Already deleted

File: first_django_project/discography/migrations/0002_alter_track_artist.py (hunk 5 above)
Comment (Voovaaa) [reply first_django_project/discography/migrations/0002_alter_track_artist.py:1]: Already deleted

**Review by Safintim — APPROVED** [review]

**Pull request merged**